### PR TITLE
TASK-25 Auto-complete for topic names

### PR DESCRIPTION
**Pull request by maxtrof**

### CODE REVIEW

File: UI/Controls/ListenerEditControl.axaml.cs [hunk 1]
```diff
@@ -32,4 +36,9 @@ private void MessageTextBlock_OnPointerPressed(object? sender, PointerPressedEve
         if (sender is TextBlock block)
             Avalonia.Application.Current?.Clipboard?.SetTextAsync(block.Text);
     }
+
+    private void TopicAutoCompleteBox_OnLostFocus(object? sender, RoutedEventArgs e)
+    {
+        MessageBus.Current.SendMessage(_autoCompleteBox.Text, BusEvents.AutocompleteAdded);
```
Comment (SlashV83) [inline UI/Controls/ListenerEditControl.axaml.cs]: sender is AutoCompleteBox here, just need to cast it
if(sender is AutoCompleteBox autoCompleteBox)
{...}

File: UI/Controls/ListenerEditControl.axaml.cs [hunk 2]
```diff
@@ -20,6 +23,7 @@ internal partial class ListenerEditControl : UserControl
     public ListenerEditControl()
     {
         InitializeComponent();
+        _autoCompleteBox = this.FindControl<AutoCompleteBox>("TopicAutoCompleteBox");
```
Comment (SlashV83) [inline UI/Controls/ListenerEditControl.axaml.cs]: search controls in constructor is usefull when we want to subscribe to controls events manually in code. And unsubscribe later, maybe

Comment (maxtrof) [reply UI/Controls/ListenerEditControl.axaml.cs]: Done

File: UI/Controls/ListenerEditControl.axaml.cs (hunk 1 above)
Comment (maxtrof) [reply UI/Controls/ListenerEditControl.axaml.cs]: Done

**Pull request merged**